src/components/ButtonAppBar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import IconButton from '@material-ui/core/IconButton';
+import { Link } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
make separate `style.js` files and import them. If you put your style in the components it gets too long